Python/Product/PythonTools/PythonTools.vsct

```diff
@@ -624,8 +624,17 @@ permissions and limitations under the License.
         </Strings>
       </Button>
 
+      <Button guid="guidPythonToolsCmdSet" id="cmdidProcessRequirementsTxt" priority="0x0610" type="Button">
+        <CommandFlag>AllowParams</CommandFlag>
+        <CommandFlag>CommandWellOnly</CommandFlag>
+        <Strings>
+          <ButtonText>Pro&amp;cess Requirements</ButtonText>
```

Comment:
Is there a more descriptive name than "process"? "InstallProjectRequirements" maybe? Can/should we switch the existing wizard to use this command as well?